docs/src/components/current-refinements.md

```diff
@@ -0,0 +1,39 @@
+---
+title: CurrentRefinements
+mainTitle: Components
+layout: main.pug
+category: Components
+withHeadings: true
+navWeight: 6
+editable: true
+githubSource: docs/src/components/current-refinements.md
+---
+
+Show the currently refined values and allow them to be unset.
+
+<a class="btn btn-static-theme" href="stories/?selectedKind=CurrentRefinements">🕹 try out live</a>
+
+## Usage
+
+```html
+<ais-current-refinements :clears-query="true"></ais-current-refinements>
+```
+
+## Props
+
+Name | Type | Default | Description | Required
+---|---|---|---|---
+attributes | Array | | Allows you to format the attributes | no
```

Comment:
should be made clearer, in InstantSearch.js too: https://community.algolia.com/instantsearch.js/v2/widgets/currentRefinedValues.html#struct-CurrentRefinedValuesWidgetOptions-attributes